CHANGELOG.md

```diff
@@ -29,13 +31,15 @@
 ### Released packages
 
 - @xen-orchestra/audit-core 0.1.1
+- @xen-orchestra/upload-ova 1.0.0
+- xo-vmdk-to-vhd 1.0.0
```

Comment:
Contrary to what has been specified, can we still do a minor for this?

Major and minor is indeed ambiguous for pre-1.0.0 versions.

Reply:
Unless it's too late, and the modules are already published, in that case forget this comment :slightly_smiling_face:

Reply:
The `xo-vmdk-to-vhd` is already published.
For `@xen-orchestra/upload-ova`, I will change the version to minor.